include/circt/Dialect/FIRRTL/FIRRTLCanonicalization.td

```diff
@@ -508,5 +510,18 @@ def StoUtoS : Pat<
   (replaceWithValue $x),
   []>;
 
+def CVTSigned : Pat<
+  (CvtPrimOp $x),
+  (replaceWithValue $x),
+  [(SIntType $x)]>;
+
+def CVTUnSigned : Pat<
+  (CvtPrimOp:$old $x),
+  (MoveNameHint $old,
+    (AsSIntPrimOp
+      (CatPrimOp 
+        (NativeCodeCall<"$_builder.create<ConstantOp>($0.getLoc(), APSInt(APInt()))"> $old),
+        $x))),
+  [(UIntType $x)]>;
```

Comment:
This canonicalization effectively removes the need for `cvt`. Maybe we just boot this from the spec or remove this during parsing and drop the operation.

Reply:
CVT primarily exists for inferred widths.  It may be convenient in that context (e.g. at chisel emission), but I'm not sure.